css/common/main.css

```diff
@@ -163,3 +144,78 @@ footer a {
 .channel-info {
   margin-left: 15px;
 }
+
+/* last screen code */
+
+.sample {
+  padding: 3rem 0;
+}
+
+.event-title {
+  width: auto;
+  font-weight: 400;
+  font-size: 2rem;
+  text-align: center;
+  color: #e30062;
+}
+
+.upcoming-event {
+  font-size: 1.5rem;
+  margin-top: 2.5rem;
+  text-align: center;
+}
+
+.api-event {
+  margin: 1.5rem 1.5rem;
+  font-size: 1.5rem;
+  border-bottom: 1px solid;
+  color: #604af0;
+}
+
+.previous-event {
+  font-size: 1.5rem;
+  margin-top: 2.5rem;
+  text-align: center;
+}
+
+.events-box {
+  margin-top: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+}
+
+.events {
+  color: #604af0;
+  border-bottom: 1px solid;
```

Comment:
no color for border?

Reply:
no, i have taken all of the anchor tags as one color.